src/utils/fetchErrors.js

```diff
@@ -0,0 +1,20 @@
+class ServerError extends Error {
+}
+
+const throwWithEndUserMessage = (response, json, friendlyMessage) => {
+  if (!response.ok) {
+    let messageToSend = json.message;
```

Comment:
I would call `messageToSend` something like `responseMessage` which make sit clearer where it comes from t oavoid confusions between the message sent to the api and the one received.

Reply:
I have gone with plain `message`. Is it ok?

Reply:
sounds good to me